src/utils/displayConceptHierarchy.tsx

```diff
@@ -0,0 +1,24 @@
+import { Fragment, ReactNode } from "react";
+
+import { ARROW_RIGHT } from "@/constants/chars";
+import { TFamilyConceptTreeNode } from "@/utils/buildConceptHierarchy";
+
+const hierarchyArrow = ` ${ARROW_RIGHT} `;
+
+export const displayConceptHierarchy = (concept: TFamilyConceptTreeNode): ReactNode => {
```

Comment:
suggestion: We've named and located this as though it's a util - but it seems to be a `FunctionComponent`? Should it not live with the components?

Reply:
True good point, moved to being a Molecule